hugo/content/contribute.md

```diff
@@ -126,14 +121,14 @@ We are thankful for any contribution on translating. Please check the {{< wiki "
 
 ### Testing
 
-To find bugs and polish new releases we need testers.
+To find bugs and polish new releases (before and after release) we need testers.
 
-If you want to test newest development versions, you have two options:
+If you want to test the newest development versions, you have two options:
 
 - We provide development snapshots. You can download them from our [Downloads page]({{< relref "downloads" >}}).
 <!-- link to exact point? https://www.mumble.info/downloads/#development-snapshots -->
 
-- Build your own version of Mumble, see [Building Mumble](https://github.com/mumble-voip/mumble#building).
+- Build the latest master of Mumble, see [Building Mumble](https://github.com/mumble-voip/mumble#building).
```

Comment:
```suggestion
- Build the latest version of Mumble (`master` branch on GitHub), see [Building Mumble](https://github.com/mumble-voip/mumble#building).
```

Reply:
Well I tried to avoid `latest version`, as that might be confusing.

What about `latest development version`?

Reply:
Sounds good to me :+1:

Reply:
What about `latest development state`?
version sounds like a release version...

I was also thinking about adding information to the building instructions about how to use specific versions (it's obvious for people who know git, but for beginners it might be useful).